src/components/common/Headline/Headline.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Headline() {
```

Comment:
What does this component do?
Just render `h1` tag?

Reply:
yes

Reply:
Where you set some text?